selfdrive/controls/lib/events.py

```diff
@@ -31,7 +31,8 @@ class Priority(IntEnum):
 class ET:
   ENABLE = 'enable'
   PRE_ENABLE = 'preEnable'
-  OVERRIDE = 'override'
+  OVERRIDE_LAT = 'overrideLat'
+  OVERRIDE_LONG = 'overrideLong'
```

Comment:
Couldn't think of a more generic way to do this. If we keep the one override event type (which I'd prefer), you can't easily use the event types for longActive anymore (`self.events.any(ET.OVERRIDE)` would be true for lat overriding). Aside from doing `and CS.brakePressed` or `and CS.steeringPressed`, but we decided against that when adding longActive for ndog.